### ISSUE TITLE
himawari-NaN.jpg

### ISSUE THREAD

**Issue by kilianc** [jakiestfu/himawari.js#15]
```
$ himawari --zoom 4 -i

Creating himawari-NaN.jpg in /Volumes/WS/workspace/foo ...
```

**Comment by jakiestfu** [comment]
What version are you using?

**Comment by kilianc** [comment]
latest, just udpdated

**Comment by jakiestfu** [comment]
This is fixed, will be pushing the new version upstream very shortly.

**Comment by jakiestfu** [comment]
Fixed in `v1.2.5`, however, please note in #16 that Infrared doesn't support zoom over 3.
